### ISSUE TITLE
configure: error: cannot import Python module "distutils"

### ISSUE THREAD

**Issue by hlovdal** [cmusphinx/pocketsphinx#242]
Steps to reproduce:

```bash
$ cd /tmp
$ git clone https://github.com/cmusphinx/pocketsphinx
Cloning into 'pocketsphinx'...
remote: Enumerating objects: 12438, done.
...
Resolving deltas: 100% (9313/9313), done.
$ cd pocketsphinx
$ ./autogen.sh --prefix=/tmp/test
processing .
Running libtoolize...
...
checking for a version of Python >= '2.1.0'... yes
checking for the distutils Python package... no
configure: error: cannot import Python module "distutils".
Please check your Python installation. The error was:
<string>:1: DeprecationWarning: The distutils package is deprecated and slated for removal in Python 3.12. Use setuptools or check PEP 632 for potential alternatives
$
```

The reference is coming from `m4/ax_python_devel.m4` which was created in 2015 and never has been modified since.

My machine is running Fedora 35.

-------

[Similar issue](https://github.com/ccontavalli/ssh-ident/issues/65).

**Comment by dhdaines** [comment]
Build system is now totally different and now you should be able to just run `pip install .` from the top-level directory.  The module name is different now though, due to rampant namespace pollution - it's "pocketsphinx5".  Try this PR if you have a moment: https://github.com/cmusphinx/pocketsphinx/pull/254

**Comment by dhdaines** [comment]
Fixed with #254
